theme/tuxlite_tbs/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```

Comment:
does this license actually permit people to use this site?  there are often issues with these tongue-in-cheek permissive licenses. @mahmoud will have a more cogent criticism i think.